CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Changelog
 
+## [2.3.2](https://launchpad.net/mixxx/+milestone/2.3.3) (unreleased)
+
+* Pioneer-DDJ-SB3: fix shift button release [#4659](https://github.com/mixxxdj/mixxx/pull/4659)
```

Comment:
What does this mean? How was it broken? What works now what didn't work before?

Also, please capitalize the first character after the colon.